### PR TITLE
Trees Pull Request

### PR DESCRIPTION
**Pull request by TheShadowOfHassen**
Ok, this is a huge pull request for all of 0 A.D.'s trees(gaia/tree) and also apples from the fruit category.

### CODE REVIEW

File: 0-ad-history-mod-package/simulation/templates/gaia/tree/poplar_lombardy_autumn.xml [hunk 1]
```diff
@@ -2,6 +2,10 @@
 <Entity parent="template_gaia_tree">
   <Identity>
     <SpecificName>Poplar</SpecificName>
+    <History>
+        Poplars(Genus Populus) are fast growing, deciduous trees of the family Salicacae. There are around 35  different kinds of the tree.
+        It is Autumn and this Poplars' leaves are fading yellow and soon will fall to the ground.
```
Comment (s0600204) [inline 0-ad-history-mod-package/simulation/templates/gaia/tree/poplar_lombardy_autumn.xml:7]: `Poplars'` -> `Poplar's`

**Review by s0600204 — COMMENTED** [review]
A lot of these read as descriptions (for which the `<Description>` tag would be better) rather than histories. Anyway...

* Be consistent with abbreviations. In some places you've used `ft`, in others, the full unit `feet`.
* Be consistent with capitalisations. e.g. `Autumn`/`autumn`, `Acorn`/`acorn`, `Summer`/`summer`, `Oak`/`oak`, `Fall`/`fall` (where it refers to the season, not something... falling).
* Use apostrophes where applicable, rather than single quotation marks. (`'` not `’`).
* The possessive form of `it` is `its`, not `it's`.
* Various entries have a missing space prior to an opening parenthesis. (`text(text` -> `text (text`).
* Various entries have a double space between words mid-sentence.

File: 0-ad-history-mod-package/simulation/templates/gaia/tree/oak_holly_autumn.xml [hunk 2]
```diff
@@ -2,6 +2,10 @@
 <Entity parent="template_gaia_tree">
   <Identity>
     <SpecificName>Holly Oak</SpecificName>
+    <History>
+        The Holly Oak(Quercus ilex)i s commonly called the Holm oak is a fascinating Oak which does not shed its leaves in winter. It grows along Southern Europe and into Asia. It grows up to 100 feet tall.
```
[the remark this answers is not in the document]
Comment (TheShadowOfHassen) [reply 0-ad-history-mod-package/simulation/templates/gaia/tree/oak_holly_autumn.xml:6]: Got It.

File: 0-ad-history-mod-package/simulation/templates/gaia/tree/pine_maritime.xml [hunk 3]
```diff
@@ -2,6 +2,9 @@
 <Entity parent="template_gaia_tree">
   <Identity>
     <SpecificName>Maritime Pine</SpecificName>
+    <History>
+       The Maritime Pine, (Pinus pinaster) as the name says, can survive in coastal locations. The Maritime Pine is even resistant to salt water. It can grow up to 130 feet, however, in exposed conditions it becomes twisted and stunted.
```
[the remark this answers is not in the document]
Comment (TheShadowOfHassen) [reply 0-ad-history-mod-package/simulation/templates/gaia/tree/pine_maritime.xml:6]: Yep you are right.

File: 0-ad-history-mod-package/simulation/templates/gaia/tree/banyan.xml [hunk 4]
```diff
@@ -6,6 +6,9 @@
   </Footprint>
   <Identity>
     <SpecificName>Banyan</SpecificName>
+    <History>
+        The Banyan Tree or Ficus benghalensis is a unique fig tree. They start as a small epiphyte that grows on a host tree before suffocating and taking it’s place. The tree has huge, low spreading branches, that often grow down to become secondary trunks. Over time, the Banyan can grown to massive sizes. Most Banyans grow around 100 feet tall, but their canopy can cover an area of 1,000 feet.
```
[the remark this answers is not in the document]
Comment (TheShadowOfHassen) [reply 0-ad-history-mod-package/simulation/templates/gaia/tree/banyan.xml:10]: Got it.

File: 0-ad-history-mod-package/simulation/templates/gaia/tree/oak_aut.xml [hunk 5]
```diff
@@ -2,6 +2,10 @@
 <Entity parent="template_gaia_tree">
   <Identity>
     <SpecificName>Oak</SpecificName>
+    <History>
+        The species Oak (Quercus) is one of the most recognizable trees on earth. There are many varieties of the tree from the more standard English Oak to the Holm Oak, which is an evergreen. While leaves, heights, and widths vary one thing stays the same, the acorn. While an Acorn’s size and shape may vary from tree to tree in every form it is the oak’s most recognizable feature.
+        It is autum now and this oak's leaves are turning colors and begining to fall. Clearly, it isn't a Holm Oak
```
[the remark this answers is not in the document]
Comment (TheShadowOfHassen) [reply 0-ad-history-mod-package/simulation/templates/gaia/tree/oak_aut.xml:7]: Got it

File: 0-ad-history-mod-package/simulation/templates/gaia/tree/temperate_autumn.xml [hunk 6]
```diff
@@ -2,6 +2,19 @@
 <Entity parent="template_gaia_tree">
   <Identity>
     <SpecificName>Deciduous Tree</SpecificName>
+    <History>
+       The once green leaves of these temperate, template trees have faded into a variety of orange and red hues. Soon the trees will shed their leaves and be ready to weather the coming winter.
```
[the remark this answers is not in the document]
Comment (TheShadowOfHassen) [reply 0-ad-history-mod-package/simulation/templates/gaia/tree/temperate_autumn.xml:6]: It's a joke. I researched 70 some trees I get to make a few jokes.

File: 0-ad-history-mod-package/simulation/templates/gaia/tree/tropic_rainforest.xml [hunk 7]
```diff
@@ -2,6 +2,9 @@
 <Entity parent="template_gaia_tree">
   <Identity>
     <SpecificName>Rainforest Tree</SpecificName>
+    <History>
+        These tree’s wide canopies of leaves will not fall off at autumn. Also these trees will be given at least 100 inches of rain a year.
```
[the remark this answers is not in the document]
Comment (TheShadowOfHassen) [reply 0-ad-history-mod-package/simulation/templates/gaia/tree/tropic_rainforest.xml:6]: Yep.

File: 0-ad-history-mod-package/simulation/templates/gaia/tree/oak.xml [hunk 8]
```diff
@@ -2,6 +2,9 @@
 <Entity parent="template_gaia_tree">
   <Identity>
     <SpecificName>Oak</SpecificName>
+    <History>
+        The species Oak (Quercus) is one of the most recognizable trees on earth. There are many varieties of the tree from the more standard English Oak to the Holm Oak, which is an evergreen. While leaves, heights, and widths vary one thing stays the same, the acorn. While an Acorn’s size and shape may vary from tree to tree in every form it is the oak’s most recognizable feature.
```
[the remark this answers is not in the document]
Comment (TheShadowOfHassen) [reply 0-ad-history-mod-package/simulation/templates/gaia/tree/oak.xml:6]: it should be acorn

File: 0-ad-history-mod-package/simulation/templates/gaia/tree/acacia.xml [hunk 9]
```diff
@@ -2,6 +2,9 @@
 <Entity parent="template_gaia_tree">
   <Identity>
     <SpecificName>Acacia</SpecificName>
+    <History>
+        Commonly known as thorn trees, the Acacia, part of the Mimosa subfamily, grows in a variety of open habitats, varying from tropical to subtropical. While 900 of the 1,200 trees live in Australia, the tree is also prevalent in Africa. The Acacia lives in dry climates so they have deep roots systems in order to draw water from the ground.
```
[the remark this answers is not in the document]
Comment (TheShadowOfHassen) [reply 0-ad-history-mod-package/simulation/templates/gaia/tree/acacia.xml:6]: Thanks, stuff like this is the most helpful. I'm changing it to tree's species

**Comment by TheShadowOfHassen** [comment]
> A lot of these read as descriptions (for which the `<Description>` tag would be better) rather than histories. Anyway...
> 
> * Be consistent with abbreviations. In some places you've used `ft`, in others, the full unit `feet`.
> * Be consistent with capitalisations. e.g. `Autumn`/`autumn`, `Acorn`/`acorn`, `Summer`/`summer`, `Oak`/`oak`, `Fall`/`fall` (where it refers to the season, not something... falling).
> * Use apostrophes where applicable, rather than single quotation marks. (`'` not `’`).
> * The possessive form of `it` is `its`, not `it's`.
> * Various entries have a missing space prior to an opening parenthesis. (`text(text` -> `text (text`).
> * Various entries have a double space between words mid-sentence.

There is a description tag?!? No one told me. I haven't looked hard, but the description tag looks like it's been used mostly for in game descriptions, so this is more history than it is gameplay relevant description.

File: 0-ad-history-mod-package/simulation/templates/gaia/tree/euro_birch.xml [hunk 10]
```diff
@@ -2,6 +2,9 @@
 <Entity parent="template_gaia_tree">
   <Identity>
     <SpecificName>Silver Birch</SpecificName>
+    <History>
+        The European Birch or Silver Birch is a hearty tree, able to survive the harsh weather in Scandinavia as the warm Mediterranean cost. It grows up to 100 ft, and its bark is white to silver and has serrated, triangular leaves.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/gaia/tree/euro_birch.xml:6]:
```suggestion
        The European Birch or Silver Birch is a hearty tree, able to survive the harsh weather in Scandinavia as the warm Mediterranean cost. It grows up to 100 feet, and its bark is white to silver and has serrated, triangular leaves. 
```

File: 0-ad-history-mod-package/simulation/templates/gaia/tree/palm_palmyra.xml [hunk 11]
```diff
@@ -2,6 +2,9 @@
 <Entity parent="template_gaia_tree">
   <Identity>
     <SpecificName>Palmyra Palm</SpecificName>
+    <History>
+        The Palmyra Palm (Borassus flabellifer) is also known by many other names. It grows up to 98 ft tall and produces palmyra fruit which is usually eaten unripe and tastes like coconut.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/gaia/tree/palm_palmyra.xml:6]:
```suggestion
        The Palmyra Palm (Borassus flabellifer) is also known by many other names. It grows up to 98 feet tall and produces palmyra fruit which is usually eaten unripe and tastes like coconut.
```

File: 0-ad-history-mod-package/simulation/templates/gaia/tree/poplar_lombardy_autumn.xml [hunk 12]
```diff
@@ -2,6 +2,10 @@
 <Entity parent="template_gaia_tree">
   <Identity>
     <SpecificName>Poplar</SpecificName>
+    <History>
+        Poplars(Genus Populus) are fast growing, deciduous trees of the family Salicacae. There are around 35 different kinds of the tree.
+        It is Autumn and this Poplar's leaves are fading yellow and soon will fall to the ground.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/gaia/tree/poplar_lombardy_autumn.xml:7]:
```suggestion
        It is autumn and this poplar's leaves are fading yellow and soon will fall to the ground.
```

File: 0-ad-history-mod-package/simulation/templates/gaia/tree/maple_autumn.xml [hunk 13]
```diff
@@ -2,6 +2,10 @@
 <Entity parent="template_gaia_tree">
   <Identity>
     <SpecificName>Maple</SpecificName>
+    <History>
+        Maple trees are instantly recognized by their unique palmate leaves. Most species of maple live in either Japan or North America. However the Norway Maple(Acer platanoides) grew from ancient times from Europe down to the northwestern corner of India.
+        It is Fall and this maple tree's leaves are turning crimson.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/gaia/tree/maple_autumn.xml:7]:
```suggestion
        It is fall and this maple tree's leaves are turning crimson.
```

File: 0-ad-history-mod-package/simulation/templates/gaia/tree/oak_holly.xml [hunk 14]
```diff
@@ -2,6 +2,9 @@
 <Entity parent="template_gaia_tree">
   <Identity>
     <SpecificName>Holly Oak</SpecificName>
+    <History>
+        The Holly Oak(Quercus ilex) is also called the Holm Oak. It is a fascinating Oak which does not shed its leaves in winter. It grows along Southern Europe and into Asia. It grows up to 100 feet tall.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/gaia/tree/oak_holly.xml:6]:
```suggestion
        The Holly Oak (Quercus ilex) is also called the Holm Oak. It is a fascinating Oak which does not shed its leaves in winter. It grows along Southern Europe and into Asia. It grows up to 100 feet tall.
```

File: 0-ad-history-mod-package/simulation/templates/gaia/tree/oak_holly_autumn.xml [hunk 15]
```diff
@@ -2,6 +2,10 @@
 <Entity parent="template_gaia_tree">
   <Identity>
     <SpecificName>Holly Oak</SpecificName>
+    <History>
+        The Holly Oak(Quercus ilex) is also called the Holm Oak. It is a fascinating Oak which does not shed its leaves in winter. It grows along Southern Europe and into Asia. It grows up to 100 feet tall.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/gaia/tree/oak_holly_autumn.xml:6]:
```suggestion
        The Holly Oak (Quercus ilex) is also called the Holm Oak. It is a fascinating Oak which does not shed its leaves in winter. It grows along Southern Europe and into Asia. It grows up to 100 feet tall.
```

**Review by indoptogopt — CHANGES_REQUESTED** [review]

**Comment by TheShadowOfHassen** [comment]
We are capitalizing seasons like Fall, Autumn and Winter, so those changes can't happen.

I think I got everything else.

Is this clear to be merged?

**Comment by indoptogopt** [comment]
Yes, you can merge it.

**Pull request merged**